### ISSUE TITLE
importing libhackrf doesnt work

### ISSUE THREAD

**Issue by menroka** [dressel/pyhackrf#5]
I cloned the git files into my project directory, but when trying to import libhackrf, the following error gets raised:

Traceback (most recent call last):
  File "C:/Users/name/.PyCharmCE2018.3/config/scratches/Bluetooth/search tree.py", line 6, in <module>
    import libhackrf as hcrf
  File "C:\Users\name\.PyCharmCE2018.3\config\scratches\Bluetooth\libhackrf.py", line 21, in <module>
    libhackrf = CDLL('libhackrf.so.0')
  File "C:\Users\name\AppData\Local\Programs\Python\Python37\lib\ctypes\__init__.py", line 356, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: [WinError 126] The specified module could not be found

Using the "from libhackrf import *" from the examples yields the same result.

**Comment by Ayoub1375** [comment]
I have the same problem

**Comment by menroka** [comment]
To anyone reading this: Don't bother using any products of Great Scott Gadgets.
I have tried to work with the hackrf one, which has terrible performance and synchronization problems and cant even send the most basic signal at the right time.
I have also tried to work with the Ubertooth, for which not even the most basic "hacking" scripts are implemented.
Not even speaking of any documentation, specification, or reference manuals.

Buying their products is just a waste of time. You are much better off using an embedded SoC, for example from Nordic, and creating an interface through SPI/UART to your PC, if absolutely necessary.
